tests/codeception/acceptance/Content/NewsCest.php

```diff
@@ -311,10 +311,24 @@ public function testRelatedNewsPerson(AcceptanceTester $I) {
    */
   #[CodeceptionAttribute\Group('news_variant')]
   public function testNewsVariantFieldsDisplay(AcceptanceTester $I) {
+    $I->logInWithRole('site_editor');
```

Comment:
A better solution is to use codeception examples.
see this example: https://github.com/SU-SWS/stanford_profile/blob/12.x/tests/codeception/acceptance/Users/RolesCest.php#L234